sdk/eventhub/azure-eventhub/migration_guide.md

```diff
@@ -0,0 +1,250 @@
+# Migration Guide (EventHubs v1 to v5)
+
+This document is intended for users that are familiar with V1 of the Python SDK for Event Hubs library (`azure-eventhub 1.x.x`) and wish 
+to migrate their application to V5 of the same library.
+
+For users new to the Python SDK for Event Hubs, please see the [readme file for the azure-eventhub](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhub/README.md).
+
+## General changes
+
+In the interest of simplifying the API surface we've made two distinct
+clients, rather than having a single `EventHubClient`. 
+* `EventHubProducerClient` for sending messages. [Sync API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-eventhub/5.0.0b6/azure.eventhub.html#azure.eventhub.EventHubProducerClient)
+and [Async API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-eventhub/5.0.0b6/azure.eventhub.aio.html#azure.eventhub.aio.EventHubProducerClient)
+* `EventHubConsumerClient` for receiving messages. [Sync API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-eventhub/5.0.0b6/azure.eventhub.html#azure.eventhub.EventHubConsumerClient)
+and [Async API](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-eventhub/5.0.0b6/azure.eventhub.aio.html#azure.eventhub.aio.EventHubConsumerClient)
+
+We've also merged the functionality from `EventProcessorHost` into 
+`EventHubConsumerClient`, allowing `EventHubConsumerClient` to be the single
+point of entry for receiving of any type (from single partition, all partitions, or with load balancing and checkpointing features) within Event Hubs.
+
+V5 has both sync and async APIs. Sync API is under package azure.eventhub whereas async API is under package azure.eventhub.aio.
```

Comment:
To be consistent, i'd also add back ticks on azure.eventhub and azure.eventhub.aio